zeebe/atomix/cluster/src/main/java/io/atomix/cluster/messaging/impl/NettyMessagingService.java

```diff
@@ -682,7 +701,9 @@ private <T> void executeOnPooledConnection(
                                   .addListener(
                                       f -> {
                                         log.debug(
-                                            "Closing connection to {}", channel.remoteAddress());
+                                            "Closing connection to {} - {}",
+                                            channel.remoteAddress(),
+                                            MDC.get("actor-scheduler"));
```

Comment:
:x: No need to add actor-scheduler to the logger. It should be already part of the logging context.

Reply:
Reverted.